pkg/cloudprovider/aws/vpc.go

```diff
@@ -28,7 +28,17 @@ import (
 )
 
 type VPCProvider struct {
+	ec2            ec2iface.EC2API
```

Comment:
maybe call this `ec2api` for consistency?

Reply:
👍  done!